src/steps/os/windows.rs

```diff
@@ -1,4 +1,4 @@
-use std::convert::TryFrom;
+// use std::convert::TryFrom;
```

Comment:
Are you on a nightly toolchain and got some redundant import warnings?

Reply:
exactly

Reply:
Can we remove it rather than commenting it if the stable toolchain is ok with this